op-e2e/actions/l2_runtime.go

```diff
@@ -49,6 +56,9 @@ type Runtime struct {
 	colosseumContract        *bindings.Colosseum
 	securityCouncilContract  *bindings.SecurityCouncil
 	valPoolContract          *bindings.ValidatorPoolCaller
+	valManContract           *bindings.ValidatorManagerCaller
```

Comment:
Just an opinion, why not shorten it to valMgr instead of valMan?

Reply:
Good idea, I'll apply it in the next PR.